java/ql/test/stubs/springframework-5.2.3/org/springframework/validation/FieldError.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2002-2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.validation;
+
+import org.springframework.lang.Nullable;
+
+public class FieldError extends ObjectError {
+
+  private final String field;
```

Comment:
This file isn't quite just stubs. Get rid of superfluous code such as private fields and constructor bodies.